pkg/controller/certificate-shim/helper_test.go

```diff
@@ -39,10 +39,23 @@ func Test_translateAnnotations(t *testing.T) {
 
 	validAnnotations := func() map[string]string {
 		return map[string]string{
-			cmapi.CommonNameAnnotationKey:  "www.example.com",
-			cmapi.DurationAnnotationKey:    "168h", // 1 week
-			cmapi.RenewBeforeAnnotationKey: "24h",
-			cmapi.UsagesAnnotationKey:      "server auth,signing",
+			cmapi.CommonNameAnnotationKey:                 "www.example.com",
+			cmapi.EmailsAnnotationKey:                     "test@example.com",
+			cmapi.SubjectOrganizationsAnnotationKey:       "Test Organization",
+			cmapi.SubjectOrganizationalUnitsAnnotationKey: "Test Organizational Unit",
+			cmapi.SubjectCountriesAnnotationKey:           "Country",
+			cmapi.SubjectProvincesAnnotationKey:           "Province",
+			cmapi.SubjectLocalitiesAnnotationKey:          "City",
+<<<<<<< HEAD
+			cmapi.SubjectStreetAddressesAnnotationKey:     "\"1725 Slough Avenue, Suite 200, Scranton Business Park\"",
+=======
+			cmapi.SubjectStreetAddressesAnnotationKey:     "Address",
+>>>>>>> 10a9535a99ece4b45f2ea7d8c4aa57bd35d52768
```

Comment:
I'm guessing these conflicts are just temporary? Obviously will need to be fixed before merging!